### DISCUSSION TITLE
Feature requests: Instead burn, expiration, visibility for every paste etc.

### DISCUSSION THREAD

**Discussion by TheZoker** [interaapps/pastefy#36] [Ideas]
Hi there,

I have some requests for some features to make the app perfect for me:

- Add option for pastes to expire (after one view aka instand burn, 5 minutes, 60 minutes etc.) then the paste will be deleted
- Add confirm for user who open a instand burn paste (like "This paste will delete itself after you viewed it. Are you sure?")
- Add option for every paste to be public, private or unlisted. Public can be listed on a pastes list, private is only accessible for the user who posted it and unlisted can be shared but won't show on the pastes list
- Internal authentication system that is based on the database (or config options if that is easier). I don't want to use an external oauth service

Looking forward to your feedback on this!

**Comment by JulianFun123** [comment]
Expiration and Visibility are now implemented. We do not plan an internal authentication system and instant burn is also not planned.
